memgraph_versioned_docs/version-2.5.2/connect-to-memgraph/drivers/java.md

```diff
@@ -77,24 +73,17 @@ public class HelloWorld implements AutoCloseable
 
     public void printGreeting( final String message )
     {
-        try ( Session session = driver.session() )
-        {
-            String greeting = session.writeTransaction( new TransactionWork<String>()
-            {
-                @Override
-                public String execute( Transaction tx )
-                {
-                    Result result = tx.run( "CREATE (a:Greeting) " +
-                                                     "SET a.message = $message " +
-                                                     "RETURN 'Node ' + id(a) + ': ' + a.message",
-                            parameters( "message", message ) );
-                    return result.single().get( 0 ).asString();
-                }
-            } );
-            System.out.println( greeting );
-        }
+    	
+    	try (var session = driver.session()) {
+    		var hello = session.executeWrite( transaction -> {
+    		var query = new Query("CREATE (a:Greeting) " + "SET a.message = $message " + "RETURN 'Node ' + id(a) + ': ' + a.message", parameters("message", message));
```

Comment:
you can remove all + here

Reply:
I don't understand what you mean :)

Reply:
sorry... `"CREATE (a:Greeting) " + "SET a.message = $message "` can be `"CREATE (a:Greeting) SET a.message = $message "`

Reply:
a saw this pattern twice

Reply:
Thanks, will fix it!